### PR TITLE
README: Fix punctuation.

### PR DESCRIPTION
**Pull request by yyu**

### CODE REVIEW

**Review by retrage — CHANGES_REQUESTED** [review]
Each commit must have a "signed-off by" signature as the DCO bot indicates.

File: README.md [hunk 1]
```diff
@@ -101,4 +103,4 @@ sha1sum: 5fc086643dea4b20c59a795a262e0d2400fab15f
 ## Security issues
 
 Please use the GitHub security advisories feature for reporting issues:
-https://github.com/cloud-hypervisor/rust-hypervisor-firmware/security/advisories/new
+https://github.com/cloud-hypervisor/rust-hypervisor-firmware/security/advisories/new.
```
[the remark this answers is not in the document]
Comment (yyu) [reply README.md]: done.

**Comment by rbradford** [comment]
This should also be squashed into one commit.

**Comment by yyu** [comment]
> This should also be squashed into one commit.

done

**Review by retrage — CHANGES_REQUESTED** [review]
Please add `Signed-off-by` to the commit.
See https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff

**Review by retrage — APPROVED** [review]
Thanks!

**Pull request closed without merging**